ringcentral/http/client.py

```diff
@@ -13,11 +13,14 @@ class Client:
     def __init__(self):
         pass
 
-    def send(self, request):
+    def send(self, request, multipart_mixed=False):
         response = None
 
         try:
-            response = self.load_response(request.prepare())
+            prepared = request.prepare()
```

Comment:
You can add a check here if the request was already prepared to skip it in this case.